### DISCUSSION TITLE
scrum prep Milestone 3 - 1

### DISCUSSION THREAD

**Discussion by MrL9000** [uprm-inso4116-2024-2025-s1/semester-project-trolley-tracker-app#150] [General]
Write your scrums here

**Comment by MrL9000** [comment]
There was no time for work to be done.

**Comment by gabrielpadilla7** [comment]
No work to be done

**Comment by Yahid1** [comment]
Task were put for milestone #3 and objectives where determined.

**Comment by marcosarraiza** [comment]
I had a couple exams this week, no work could be done with the time at hand.

**Comment by JoelTorres20** [comment]
I created a task to fix a bug in the code, apart from that, I didn't have enough time to do anything else.

**Comment by AlexisTraversoPR** [comment]
This week I had no time to really work on the project

**Comment by sebasagir** [comment]
I achieved unprecedented success in doing nothing

**Comment by yamayrie** [comment]
No work was done

**Comment by dylanDeJesus21** [comment]
Studied code for possible future tasks

**Comment by dAninAd1248** [comment]
Did absolutely nothing

**Comment by Alej24** [comment]
No tasks were done.

**Comment by JConcepcion01** [comment]
no work done this week yet

**Comment by Janiel777** [comment]
No hice ninguna tarea está semana

**Comment by JaseyAcevedo** [comment]
Thinking about ltt and working on a task

**Comment by IvanierB** [comment]
No tasks were done.
